src/features/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import './_404.scss';
+
+const NotFound: React.FC = (): JSX.Element => {
+	return (
+		<section className="error__section section--padding">
+			<div className="container">
+				<div className="row row-cols-1">
+					<div className="col">
+						<div className="error__content text-center">
+							<img
+								className="error__content--img mb-50"
+								src="./src/img/other/404-thumb.png"
```

Comment:
use `import image-404 from ''./src/img/other/404-thumb.png"`

Reply:
Do you have other 404 pages? (404-somethings)

Reply:
fixit